Portfolio: setup and mobile version skeleton

abelagza opened the pull request:
# This is The Portfolio mobile version skeleton.

#### In the milestone I:

* Added instructions to the ReadMe file
* Set up the local environment
* Created the HTML file and added the text.
* Styled the page using CSS
* Fixed errors

https://www.loom.com/share/b60f8e3825cb434e91d73f92ca3b67a9

abelagza committed:
Add instructions to readme file

abelagza committed:
test commit

abelagza committed:
Set up local environment

abelagza committed:
Create hambuger button

abelagza committed:
Create burger button

abelagza committed:
Create projects section

abelagza committed:
Fixed linter errors

abelagza committed:
Added live demo to the readme file

smunozmo reviewed (changes requested):
Hi @abelagza , 👋  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not pass the linters check.

![Captura de pantalla 2022-03-16 a la(s) 6 59 15 p  m](https://user-images.githubusercontent.com/83717285/158710903-00cf3950-0910-4aca-94a2-531935e6e96a.png)


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

abelagza committed:
fix linter errors

omar-muhamad commented on index.html:
```diff
@@ -0,0 +1,188 @@
+        <a href="#" class="letsconnect">LET'S CONNECT</a>
+    </div>
+    
+    <div class="icons">
+        <a href="#"><img src="./images/icons/twitter.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/linkedin.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/github.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/medium.png" alt="" class="icon"></a>
+        <a href="#"><img src="./images/icons/v.png" alt="" class="icon"></a>
+    </div>
```
-[ ] Please make sure that social icons use a `<ul>` tag to group them, and `<li>` per icon, and each icon has a link (`<a>` tag).

Kingstalux replied:
resolved 👍

omar-muhamad commented on index.html:
```diff
@@ -0,0 +1,188 @@
+    </div>
+    
+    <div class="button-c">
+        <a href="#" class="letsconnect">LET'S CONNECT</a>
```
Please add `LET'S CONNECT` inside `<h2>` or `<h3>` instead of `<a>` tag.

Kingstalux replied:
not resolved

abelagza replied:
I just figured it out.

omar-muhamad commented on style.css:
```diff
@@ -0,0 +1,421 @@
+/* Mobile version */
+
+@media only screen and (max-width: 768px) {
+  * {
+    margin: 0%;
+  }
+
+  header .nav-bar {
+    background-color: white;
+    padding: 30px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  header .my-logo {
+    font-weight: 700;
+    color: #6070ff;
+    text-decoration: none;
+    font-size: 20px;
+  }
+
+  header .nav-bar .button-b {
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
```
- [ ] Please Fix your style because when you choose a narrow mobile display mode the page content starts to overflow.
You can see this [article](https://www.w3schools.com/css/css3_box-sizing.asp) I think it will help you. 

![image](https://user-images.githubusercontent.com/68796232/158720856-f2d4efad-e85d-4dc0-876c-e9c13331f230.png)

omar-muhamad reviewed (changes requested):
# Changes Required ♻️
### Hi @abelagza,

Nice job, you are almost there.

![](https://media.giphy.com/media/UdaSUYClr92cjWpeVx/giphy.gif)

## (Highlights) Good Points: :+1:
- You documented your work professionally. :heavy_check_mark:
- All linters checks are ok. :heavy_check_mark:
- Github flow is used correctly. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- [ ] Please remove unused sections inside your Readme file like (Install, Usage, Run tests, and development) to make your Readme file more professional.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

abelagza committed:
Remove unused sections from the readme file

abelagza committed:
Remove noused sections from the readme file

abelagza committed:
Move icons into an UL tag

abelagza committed:
Add shadow effects to the title

abelagza committed:
Add mobile responsiveness

abelagza commented:
# This is The Portfolio mobile version skeleton.
#### In this milestone I:
- Moved social icons into a UL tag
- Removed non-used sections from the readme file
- Moved the "Let's connect" into a paragraph tag, h2 or h3 weren't an option as it bolds the text.
- Implemented responsiveness for devices with <400px screens.

abelagza committed:
increase body height to 80vh

Kingstalux commented on index.html:
```diff
@@ -5,132 +5,179 @@
     </div>
+    
     <div class="button-c">
         <a href="#" class="letsconnect">LET'S CONNECT</a>
```
- [ ] The text `Let's Connect` is a heading and as a result, should be placed in an `hx` tag like an `h2` or `h3` tag for best semantic practices. Kindly rectify this issue 👍

abelagza committed:
Move let's connect button into a h2 tag

Kingstalux commented on index.html:
```diff
@@ -5,132 +5,179 @@
         <img src="./images/projects1.png" alt="">
     </div>
     <div class="project-title">
         <h2>Tonic</h2>
     </div>
     <div class="subtitle">
-        <p>CANOPY • Back End Dev • 2015</p>
+        <p>CANOPY <span class="dot">•</span> <span class="backend">Back End Dev</span> <span class="dot">•</span> <span class="num">2015</span></p>
     </div>
     <div class="project-text">
-        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
     </div>
-    <div class="lan-tags">
+    <div class="lan-tags">     
+    <div class="tag-button">
         <p class="html">html</p>
```
- [ ] As mentioned before, this section is not necessary for this project so therefore it would be best to remove it 💯

Kingstalux reviewed (changes requested):
Hi @abelagza , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
![68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f7741366159414f336a4836526d55494259732f67697068792e676966](https://user-images.githubusercontent.com/85113567/158865038-44ec2be6-9a14-43a2-8f05-70d8fe2e8be8.gif)


 ## Required Changes ♻️
_Check the comments under the review._
- [ ] (OPTIONAL) It would be great if you could centralize your design. Right now it looks okay but it is leaning towards the left side. 
- Your design:
![Annotation 2022-03-17 184536](https://user-images.githubusercontent.com/85113567/158864986-635f0fb1-e7ff-4c08-a519-90547dbb4547.jpg)

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abelagza committed:
Correct mobile responsiveness

abelagza commented:
# This is The Portfolio mobile version skeleton.
#### In this milestone I:
- Moved the "Let's connect" into h2 tag.
- Fixed the left-leaning error for screens <400px.

karvel-code reviewed (approved):
Hi @abelagza , 

## Status: APPROVED 💯 🥇 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

karvel-code reviewed (commented):
- [ ] (OPTIONAL) I would advise improving on the alignment of your design as that of the figma design.
